src/SourceCompile/AnalyzeFile.cpp

```diff
@@ -125,7 +125,7 @@ std::string AnalyzeFile::setSLlineDirective_(unsigned int lineNb,
     result += std::to_string(1);
     result += "\n";
   }
-  result = "";
+  result = "";  // BUG or intentional ?
```

Comment:
here I have a question: independent of what happens above, here, the result is cleared out again.
Is this intentional or the leftover from some previous debugging ?